swift/test/Ice/objects/TestI.swift

```diff
@@ -250,15 +250,16 @@ class F2I: F2 {
     func op(current _: Current) async throws {}
 }
 
-class UnexpectedObjectExceptionTestI: Ice.Blobject {
-    func ice_invoke(inEncaps _: Data, current: Ice.Current) throws -> (ok: Bool, outParams: Data) {
+class UnexpectedObjectExceptionTestI: Ice.Dispatcher {
+    public func dispatch(_ request: IncomingRequest) async throws -> OutgoingResponse {
+        let current = request.current
         let communicator = current.adapter.getCommunicator()
```

Comment:
Same comment as for the invoke test: better let makeOutgoingResponse create the OutputStream.